packages/amplication-client/project.json

```diff
@@ -15,6 +15,7 @@
       "executor": "@nrwl/webpack:webpack",
       "outputs": ["{options.outputPath}"],
       "defaultConfiguration": "production",
+      "dependsOn": ["prebuild"],
```

Comment:
This is redundant since all `build` targets dependsOn `prebuild` targets of the same project. 
You can see the default target dependency in the nx graph or just checking the `nx.json`